core/pom.xml

```diff
@@ -13,7 +13,7 @@
 
   <artifactId>jaxen</artifactId>
   <packaging>bundle</packaging>
-  <name>jaxen</name>
+  <name>${project.artifactId}</name>
```

Comment:
That these are the same is a coincidence. They are not semantically equivalent and shouldn't be linked with a variable.

Reply:
This change is not needed, i was just lost about directory names, artifactIds and `<name>` names...